### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by felipellrocha**
See the issue here: https://github.com/expo/expo-cli/issues/1852#issuecomment-698057013

The build wasn't working because of a weird misconfiguration. This will
fix the build by default until Expo can fix things on their end.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "start": "expo start",
     "android": "expo start --android",
-    "ios": "expo start --ios",
+    "ios": "EXPO_USE_DEV_SERVER=true expo start --ios",
```
Comment (erinkav) [inline package.json:7]: we should make sure we remove this for the production build but otherwise LGTM

**Review by erinkav — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (felipellrocha) [reply package.json:7]: Mmmmm, true. Didn't think of it. Problem is that expo is *only* building when the variable is set. I'll reach out to the team. 

Alternatively, we could just ask people to add this `var` to their `.bashrc`/`.zshrc`. Wdyt?

Comment (erinkav) [reply package.json:7]: this is fine short term for development, i'd rather not ask everyone to set env variables right now. Curious what the expo team says about the production build though, im curious why the dev server is required for a build

Comment (felipellrocha) [reply package.json:7]: Cool. Will merge it, then. Yeah, I did ask, but it seems like the guy with the context is not currently online. I'll ping them tomorrow to get more context.

**Pull request merged**